infra/terraform/modules/data_access/teams/teams.py

```diff
@@ -38,5 +38,96 @@ def create_team_bucket(event, context):
     )
 
 
+def create_team_bucket_policies(event, context):
+    """
+    Creates the policies for the team S3 bucket:
+      - a "readonly" policy and
+      - a "readwrite" policy
+
+    event = {"team": {"slug": "justice-league"}}
+    """
+    bucket = bucket_name(event["team"]["slug"])
+
+    create_policy(
+        name="{}-readonly".format(bucket),
+        bucket=bucket,
+        readwrite=False,
+    )
+
+    create_policy(
+        name="{}-readwrite".format(bucket),
+        bucket=bucket,
+        readwrite=True,
+    )
+
+
+def create_policy(name, bucket, readwrite):
+    LOG.debug("Creating '{}' policy".format(name))
+
+    policy_document = get_policy_document(bucket, readwrite)
+
+    client = boto3.client("iam")
+    client.create_policy(
+        PolicyName=name,
+        Path="/teams/",
+        PolicyDocument=json.dumps(policy_document),
+    )
+
+
+def get_policy_document(bucket, readwrite):
+    bucket_arn = "arn:aws:s3:::{}".format(bucket)
+
+    statements = [
+        {
+            "Sid": "ListBucketsInConsole",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetBucketLocation",
+                "s3:ListAllMyBuckets"
+            ],
+            "Resource": "arn:aws:s3:::*"
+        },
+        {
+            "Sid": "ListObjects",
+            "Action": [
+                "s3:ListBucket"
+            ],
+            "Effect": "Allow",
+            "Resource": [bucket_arn],
+        },
+        {
+            "Sid": "ReadObjects",
+            "Action": [
+                "s3:GetObject",
+                "s3:GetObjectAcl",
+                "s3:GetObjectVersion",
+            ],
+            "Effect": "Allow",
+            "Resource": "{}/*".format(bucket_arn)
+        },
+    ]
+
+    if readwrite:
```

Comment:
If you passed in `name` rather than `readwrite`, this test could be `if name.endswith('-readwrite')`, which is still easy to understand.